auth-service/src/main/kotlin/uk/gov/communities/delta/auth/tasks/UpdateUserGUIDMap.kt

```diff
@@ -3,60 +3,34 @@ package uk.gov.communities.delta.auth.tasks
 import org.jetbrains.annotations.Blocking
 import org.slf4j.LoggerFactory
 import uk.gov.communities.delta.auth.config.LDAPConfig
-import uk.gov.communities.delta.auth.repositories.*
+import uk.gov.communities.delta.auth.repositories.DbPool
+import uk.gov.communities.delta.auth.repositories.LdapRepository
+import uk.gov.communities.delta.auth.repositories.getNewModeObjectGuidString
+import uk.gov.communities.delta.auth.repositories.searchPaged
 import java.sql.Connection
+import java.util.*
 import javax.naming.directory.SearchControls
 import kotlin.time.Duration.Companion.minutes
 
 /*
- * Update the user_guid_map database table by reading all users from Active Directory in both old and new GUID mode.
- * The migration to the new GUID format in Delta is now complete so this job is no longer run regularly,
- * though we'll keep the code and table around for a short while.
+ * Update the user_guid_map table to contain latest up to date userGUIDs and userCNs from Active Directory
```

Comment:
put comment to never run this on prod again, would lose any users deleted from ad